javascript/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@edgepi-cloud/rpc-protobuf",
   "version": "1.0.18",
```

Comment:
I think we should keep all the versions for all the languages consistent. when  there is a change in proto files we will generate language specific files and update all the version numbers. we don't have generated files for js, but we should keep the version number consistent.